android/source/VideoLocker/src/org/edx/mobile/model/registration/RegistrationDescription.java

```diff
@@ -0,0 +1,14 @@
+package org.edx.mobile.model.registration;
+
+import java.util.List;
+
+public class RegistrationDescription {
+    // We should use this to make the actual registration request, though it will be very surprising
+    // if it’s different from the endpoint we use to GET the description.
+    String endpoint;
+
+    // Similarly, we should use this method, though it will almost certainly be always be POST.
+    // If methods dynamically is a problem, it’s okay to ignore it.
```

Comment:
```
// If methods dynamically is a problem, it’s okay to ignore it.
```

This part of the comment doesn't need to be in the code. It was just a guide for implementation.